include/deal.II/grid/cell_id_translator.h

```diff
@@ -54,6 +54,11 @@ namespace internal
   class CellIDTranslator
   {
   public:
+    /**
+     * Default constructor.
+     */
+    CellIDTranslator();
```

Comment:
For computation of the level index, one does not need to specify the number coarse-cells and refinements.